tests/request/copy-request/keyboard-shortcuts.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { test, expect } from '../../../playwright';
+import { closeAllCollections, createCollection } from '../../utils/page';
+
+test.describe('Copy and Paste with Keyboard Shortcuts', () => {
+  test.afterAll(async ({ page }) => {
+    await closeAllCollections(page);
+  });
+
+  test('should copy and paste request using keyboard shortcuts', async ({ page, createTmpDir }) => {
+    await createCollection(page, 'keyboard-test', await createTmpDir('keyboard-test'), { openWithSandboxMode: 'safe' });
+    const collection = page.locator('.collection-name').filter({ hasText: 'keyboard-test' });
+
+    // Create a request
+    await collection.locator('.collection-actions').hover();
+    await collection.locator('.collection-actions .icon').click();
+    await page.locator('.dropdown-item').filter({ hasText: 'New Request' }).click();
+    await page.getByPlaceholder('Request Name').fill('test-request');
+    await page.locator('#new-request-url .CodeMirror').click();
+    await page.locator('textarea').fill('https://echo.usebruno.com');
+    await page.getByRole('button', { name: 'Create' }).click();
+
+    const requestItem = page.locator('.collection-item-name').filter({ hasText: 'test-request' });
+    await expect(requestItem).toBeVisible();
+
+    // Focus the request item
+    await requestItem.click();
+    await requestItem.focus();
+
+    // Wait for keyboard focus indicator
+    await expect(requestItem).toHaveClass(/item-keyboard-focused/);
+
+    // Use Cmd+C on Mac, Ctrl+C on Windows/Linux
+    const modifier = process.platform === 'darwin' ? 'Meta' : 'Control';
+    await page.keyboard.press(`${modifier}+KeyC`);
+
+    // Verify copy success (toast message)
+    await expect(page.getByText(/copied to clipboard/i).first()).toBeVisible();
+
+    // Focus the collection to paste
+    await collection.click();
+    await collection.focus();
+
+    // Use Cmd+V on Mac, Ctrl+V on Windows/Linux
+    await page.keyboard.press(`${modifier}+KeyV`);
+
+    // Verify paste success
+    await expect(page.getByText(/pasted successfully/i).first()).toBeVisible();
+    await page.waitForTimeout(1000);
+
+    // Verify the pasted request appears
+    await expect(page.locator('.collection-item-name').filter({ hasText: 'test-request' })).toHaveCount(2);
```

Comment:
```suggestion
await expect(page.locator('.collection-item-name').filter({ hasText: 'test-request' })).toHaveCount(2, { timeout: 5000 })
```

Avoid adding static waits instead let the assertion poll for it
- reduces the time the test takes to execute 
- easier to find slower parts of the app